tunnel/config.go

```diff
@@ -74,6 +82,27 @@ func (r SSHConfigFile) getHostname(host string) string {
 	return hostname
 }
 
+func (r SSHConfigFile) getLocalForward(host string) (*LocalForward, error) {
+	var local, remote string
+
+	c, err := r.sshConfig.Get(host, "LocalForward")
+	if err != nil {
+		return nil, err
+	}
+
+	l := strings.Fields(c)
+
+	if len(l) < 2 {
+		return nil, fmt.Errorf("bad forwarding specification on ssh config file")
```

Comment:
* Change the log level to warning since it is an error but the application will continue to work
* Adding the invalid configuration to the log message

```suggestion
		return nil, fmt.Warningf("bad forwarding specification on ssh config file: %s", l)
```